examples/dds-with-ray.py

```diff
@@ -4,7 +4,12 @@
 
 DEAL_COUNT = 50
 # endplay library dependency
-runtime_env = {"pip": ["endplay==0.4.11b0"]}
```

Comment:
why did we drop "==0.4.11b0"
I don't know what is in the newer versions, and I thought having the explicit version protects our example from potential future incompatibilities :thinking: in the future ?

Reply:
Whoops! Forget in battle. We should pin that version.